packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-result-getters/handlers/attachment-query-result-getter.handler.ts

```diff
@@ -21,9 +22,13 @@ export class AttachmentQueryResultGetterHandler
       workspaceId: workspaceId,
     });
 
+    const isPublic = isPublicAttachmentType(attachment.type);
+
+    const fullPath = `${process.env.SERVER_URL}/files/${attachment.fullPath}?${!isPublic ? `token=${signedPayload}` : `workspaceId=${workspaceId}`}`;
```

Comment:
hansmer chli more readable vorgstellt :D anschienend nöd wüklich. haha

other idea:

```suggestion
    const queryParams = !isPublic ? `token=${signedPayload}` : `workspaceId=${workspaceId}`;
```